e2-studio-star-rx72n-firmware/tests/test_rx_frame.c

```
@@ -2240,6 +2240,143 @@ void test_roundtrip_reset(void)
+ * - Return value is k_rx_ok
+ * - bytes_discarded == 1 (the junk byte)
+ * - Decoded frame matches the original ACK frame
+ *
+ * This exercises the k_rx_err_protocol_error → internal_find_sync_offset() →
+ * retry decode path.
+ */
+void test_resync_dropped_byte_recovery(void)
+{
+  rx_frame_t frame;
+  rx_frame_t decoded;
+  uint8_t    wire[k_frame_max_size + k_resync_prefix_len];
```

Clang (14.0.6)

[warning] variable 'wire' is not initialized

(cppcoreguidelines-init-variables)